packages/tools/src/enums/ChangeTypes.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * ChangeTypes defines the types of changes occurring on annotation and
+ * segmentation data.
+ */
+enum ChangeTypes {
```

Comment:
This is enormously helpful for all sorts of detection of types of changes.  I think it is a good overall change.  The question is where and how to document it?